bridge_ui/src/contexts/AlephiumWalletContext.tsx

```diff
@@ -101,12 +86,13 @@ export const AlephiumWalletProvider = ({
         setAccount(account)
         setAlephiumWindowObject(windowAlephium)
 
-        windowAlephium.on("addressesChanged", (_) => {
-          // TODO: await windowAlephium.getSelectedAccount()
-        })
-
-        windowAlephium.on("networkChanged", (_) => {
-          // TODO: await windowAlephium.getSelectedAccount()
+        windowAlephium.on("addressesChanged", async (addresses: string[]) => {
```

Comment:
Not sure how to handle the `networkChange` event here, since the network is fixed I suppose for the token bridge :thinking: 
If someone updated the network in `WalletConnect`, I think the session will be restarted. If someone updated the network in extension wallet, session might continue and perhaps they will just get weird errors down the line. 
Maybe we should alert the user somehow when extension wallet changes to a network that doesn't match the network supported by the deployed bridge? WDYT?

Reply:
Sounds great 👍